build.gradle

```diff
@@ -316,7 +316,7 @@ tasks.withType(Javadoc) {
 		// fixtures, and we want to have mockito also available in
 		// test, so we use API here.
 		testFixturesApi "org.mockito:mockito-core:${mockitoVersion}"
-		
```

Comment:
Unrelated whitespace change, please remove this from the PR (but feel encouraged to add it as new PR).